pycloudlib/gce/cloud.py

```diff
@@ -20,14 +22,18 @@
 
 logging.getLogger('googleapiclient.discovery').setLevel(logging.WARNING)
 
+DEFAULT_CREDENTIALS_PATH = Path.home() / '.config' / 'gcloud' / \
+    'application_default_credentials.json'
```

Comment:
I'm not 100% sure that this is the correct path: it does exist on my desktop, but if I initialise a new `gcloud` inside a container (to the point where I can list my instances), I don't see it:

```
# tree .config/gcloud/
.config/gcloud/
├── access_tokens.db
├── active_config
├── cache
│   └── daniel.watkins@canonical.com
├── config_sentinel
├── configurations
│   └── config_default
├── credentials.db
├── gce
├── legacy_credentials
│   └── daniel.watkins@canonical.com
│       └── adc.json
└── logs
    └── 2021.01.08
        ├── 17.27.25.441108.log
        ├── 17.27.37.404385.log
        ├── 17.27.41.272001.log
        ├── 17.28.11.613997.log
        ├── 17.28.16.418870.log
        ├── 17.28.24.243745.log
        └── 17.28.26.915831.log
```

`adc.json` _does_ look like the same file format (and `adc` is presumably an abbreviation for `application_default_credentials`?).